Kitodo/src/main/java/org/goobi/production/cli/helper/WikiFieldHelper.java

```diff
@@ -27,10 +27,14 @@
     private static final String TAG_INFO = "<font color=\"#0033CC\">";
     private static final String TAG_DEBUG = "<font color=\"#CCCCCC\">";
     private static final String TAG_USER = "<font color=\"#006600\">";
-    private static final String ENDTAG = "</font>";
+    private static final String TAG_END = "</font>";
 
     private static final String BREAK = "<br/>";
 
+    private WikiFieldHelper() {
```

Comment:
Why did you add this empty method?

Reply:
It is private constructor to exclude possibility of class instantiating. Everything in this class is static.